medusa/metadata/wdtv.py

```diff
@@ -162,12 +150,11 @@ def get_season_poster_path(show_obj, season):
                 break
 
         if not season_dir:
-            logger.log(u'Unable to find a season directory for season {season_num}'.format
-                       (season_num=season), logger.DEBUG)
+            log.debug(u'Unable to find a season directory for season {0}', season)
             return None
 
-        logger.log(u'Using {path}/folder.jpg as season dir for season {season_num}'.format
-                   (path=season_dir, season_num=season), logger.DEBUG)
+        log.debug(u'Using {path}/folder.jpg as season dir for season {x}',
+                  {'path': season_dir, 'x': season})
```

Comment:
In that case ```x``` and ```y``` make much more sense indeed. I think we should decide depending on the context.